app/assets/javascripts/app/views/help_view.js

```diff
@@ -133,6 +140,23 @@ app.views.Help = app.views.StaticContentView.extend({
     this.$('#faq').append(help_section.render().el);
   },
 
+  /**
+   * Returns The section title whose data-section property equals the given query
+   * Returns null if nothing found
+   * @param dataValue Value for the data-section to find
+   * @returns {jQuery}
+   */
+  findSection: function(dataValue){
```

Comment:
You can easily name it `section_reference` or `section_id` if that's what it really is or rename `section` to `section_html` or even just `html` since that's what it contains iirc.